.github/workflows/ci.yml

```
@@ -0,0 +1,24 @@
+name: CICD
```

I think we can just call it CI, not really any CD going on here
```suggestion
name: CI
```